task3-4/task3-4/BirthdayScreenList/ViewWithBirthdays.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ViewWithBirthdays.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 25.09.2022.
+//
+
+import UIKit
+
+/// Представление отображающее информацию о контакте и его дне рождения.
+final class ViewWithBirthDays: UIView {
+    
+    // MARK: - private properties
+    
+    private let avatarImageView: UIImageView
+    private let nameLable: UILabel
```

Comment:
```suggestion
    private let nameLabel UILabel
```
окончание такое же как в типе Label

Reply:
ниже везде поправь